### PR TITLE
Travis can't install go vet anymore

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
[merge]

File: hack/install-tools.sh [hunk 1]
```diff
@@ -9,12 +9,12 @@ source "${OS_ROOT}/hack/common.sh"
 GO_VERSION=($(go version))
 echo "Detected go version: $(go version)"
 
-go get golang.org/x/tools/cmd/cover github.com/tools/godep golang.org/x/tools/cmd/vet
+go get golang.org/x/tools/cmd/cover github.com/tools/godep
 
 # Check out a stable commit for go vet in order to version lock it to something we can work with
 pushd $GOPATH/src/golang.org/x/tools >/dev/null 2>&1
   git fetch
```
Comment (liggitt) [inline hack/install-tools.sh]: Delete this whole chunk? And the install of govet below?

**Comment by liggitt** [comment]
Also see 1.4 and 1.5/1.6 differences in govet raised in https://github.com/openshift/origin/pull/8407

File: hack/install-tools.sh [hunk 2]
```diff
@@ -9,12 +9,12 @@ source "${OS_ROOT}/hack/common.sh"
 GO_VERSION=($(go version))
 echo "Detected go version: $(go version)"
 
-go get golang.org/x/tools/cmd/cover github.com/tools/godep golang.org/x/tools/cmd/vet
+go get golang.org/x/tools/cmd/cover github.com/tools/godep
```
Comment (liggitt) [inline hack/install-tools.sh]: Is cover gone as well?

File: hack/install-tools.sh (hunk 1 above)
Comment (smarterclayton) [inline hack/install-tools.sh]: We need this chunk.  To install govet.

File: hack/install-tools.sh (hunk 2 above)
Comment (smarterclayton) [inline hack/install-tools.sh]: No.

**Comment by smarterclayton** [comment]
I see the comment about it not being needed, trying that first.

**Comment by smarterclayton** [comment]
[test]

**Pull request merged**